index.html

```diff
@@ -1,55 +1,32 @@
 <!DOCTYPE html>
 <html lang="ja">
+
 <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
     <title>javascript</title>
     <meta name="description" content="site">
-<!-- <link rel="stylesheet" href="css/style.css"> -->
+    <!-- <link rel="stylesheet" href="css/style.css"> -->
 
 </head>
-課題１－2
-
-
-
-
-<script>
-'use strict';
-
-let question = prompt("日本の首都は？");
-while(question !== "東京"  ) {
-    alert("不正解");
-    let question = prompt("日本の首都は？");
-
-    if (question == "東京"){
-    alert("正解です"); 
-    break;
-    }
-
-}
-
-
-
-
-
-// var question = prompt("日本の首都は?");
-// while(question == "東京") {
-//         alert("正解です"); 
-           
-//     }
-// if  (question !== "東京") {
-//     alert("不正解");
-//     prompt("日本の首都は?");
-
-//     }
-
-    
-
-</script>
-
-<!-- <script src="https://code.jquery.com/jquery-3.3.1.js"></script> -->
-<!-- <script src="js/main.js"></script> -->
-
+<body>
+    課題１－2
+    ご確認よろしくお願いします。
+    <script>
+        'use strict';
+        let question = prompt("日本の首都は？");
+        while (question !== "東京") {
+            alert("不正解");
+            let question = prompt("日本の首都は？");
```

Comment:
この変数のスコープ内で、内容が書き換わる可能性は無いと思いますので
const の方がよいと思いました。
```suggestion
            const question = prompt("日本の首都は？");
```